build.sbt

```diff
@@ -1,18 +1,44 @@
+import Dependencies._
+
+lazy val commonScalacOptions = Seq(
```

Comment:
Thanks. How about just adding the tpolecat plugin?

Reply:
I heard people having issues with that plugin in a multi-module setup but I haven't tried it out myself. Maybe worth a try.